sklearn/model_selection/_search_successive_halving.py

```diff
@@ -66,6 +68,22 @@ class BaseSuccessiveHalving(BaseSearchCV):
     Zohar Karnin, Tomer Koren, Oren Somekh
     """
 
+    _parameter_constraints: dict = {
+        **BaseSearchCV._parameter_constraints,
+        # overwrite `scoring` since multi-metrics are not supported
+        "scoring": [StrOptions(set(get_scorer_names())), callable, None],
+        "random_state": ["random_state"],
+        "max_resources": [Integral, StrOptions({"auto"})],
```

Comment:
Can max_resources be < 0 ?
For min_resources you put an interval [0, inf]. Why didn't you do the same here ?